dls-controls/malcolmjs
# Feature/table panel #161

## Description

Hooked table component into middle panel, added scroll ability for when table is longer than page height, added local state for table & ability to send PUTs to malcolm and wrote some unit tests.
Also altered middle panel container to select display based on widget tag of primary attribute rather than taking special cases of attribute name (i.e 'layout'). 

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/gui/PANDA/layoutButATable 

## Agile board tracking

connect to #161 
connect to #162
